guides/04_additional-features/03_streaming-inputs.md

```diff
@@ -0,0 +1,59 @@
+# Streaming inputs
+
+In the previous guide, we covered how to stream a sequence of outputs from an event handler. Gradio also allows you to stream images from a user's camera or audio chunks from their microphone **into** your event handler. This can be used to create real-time object detection apps or conversational chat applications with Gradio.
+
+Currently, the `gr.Image` and the `gr.Audio` components support input streaming via the `stream` event.
+Let's create the simplest streaming app possible, which simply returns the webcam stream unmodified.
+
+$demo_streaming_simple
```

Comment:
I don't think we should be putting the code within the Gradio demo, it just makes things harder to maintain. We can perhaps change how $demo is rendered to include the code but that should happen at the website level

Reply:
Yeap tracked here: https://huggingface.slack.com/archives/C03BHNJCBC4/p1724261743661269?thread_ts=1724256882.668899&cid=C03BHNJCBC4

Reply:
Added back separate code